packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -358,3 +359,32 @@ function createNativeOrder(fill: CollapsedFill): OptimizedMarketOrder {
         ...(fill as NativeCollapsedFill).nativeOrder,
     };
 }
+
+export function createSignedOrdersFromRfqtIndicativeQuotes(
+    quotes: RfqtIndicativeQuoteResponse[],
+): SignedOrderWithFillableAmounts[] {
+    return quotes.map(quote => {
+        return {
+            fillableMakerAssetAmount: quote.makerAssetAmount,
+            fillableTakerAssetAmount: quote.takerAssetAmount,
+            makerAssetAmount: quote.makerAssetAmount,
+            takerAssetAmount: quote.takerAssetAmount,
+            makerAssetData: quote.makerAssetData,
+            takerAssetData: quote.takerAssetData,
```

Comment:
super nit (feel free to leave as-is) but for the values that we are passing over verbatim I'd do:

```
const { makerAssetAmount, takerAssetAmount , makerAssetData, takerAssetData } = quote;
return {
   makerAssetAmount,
   takerAssetAmount,
   makerAssetData,
   takerAssetData,
   <rest of obj>
}
```